lib/ancestry/has_ancestry.rb

```diff
@@ -44,16 +44,14 @@ def has_ancestry options = {}
     scope :subtree_of, lambda { |object| where(to_node(object).subtree_conditions) }
     scope :siblings_of, lambda { |object| where(to_node(object).sibling_conditions) }
     scope :ordered_by_ancestry, lambda {
-      if %w(mysql mysql2 sqlite postgresql).include?(connection.adapter_name.downcase) &&
-        defined?(ActiveRecord.version) && ActiveRecord.version.to_s >= "5"
+      if %w(mysql mysql2 sqlite postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
We check rails 5, but technically it is the arel version that comes with rails 5.

Huh. we should only need to check this for the extra parameter in `matches`. Odd it ended up here.